apm-server/templates/ingress.yaml

```diff
@@ -12,23 +13,25 @@ metadata:
     {{- range $key, $value := .Values.labels }}
     {{ $key }}: {{ $value | quote }}
     {{- end }}
-{{- with .Values.ingress.annotations }}
-  annotations:
-{{ toYaml . | indent 4 }}
-{{- end }}
 spec:
+  {{- if .Values.ingress.className }}
```

Comment:
`ingressClassName` is new in 1.18 it should be added only when K8S > 1.18 using `{{- if semverCompare ">=1.18-0" .Capabilities.KubeVersion.Version }}`.

https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/